operations/bills.md

```diff
@@ -202,6 +278,17 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 | `Iban` | string  | optional | Enterprise IBAN (International Bank Account Number). |
 | `Bic` | string  | optional | Enterprise BIC (Bank Identifier Code). |
 
+#### Address
```

Comment:
@MikeAdamsMews How do we document addresses? Surely they're all the same - using the GeneralAPI dto.

Reply:
@KaliCZ the plan is to introduce a new page for 'common object definitions' and it sounds like Address would be a candidate for that. It's ready to go but just waiting for the right opportunity - there is too much activity on the docs at the moment! For now, you can just reference Address wherever it is defined in the docs, e.g. configuration.md#address (assuming of course that this is the same object definition).

Reply:
I have dug around and unfortunately I haven't found any suitable existing dto that would suffice here. I have found an Address defined in configuration.md and also an Address defined in addresses.md. Neither of them match the same object definition